### ISSUE TITLE
The link "Resend verification email" is displayed even if require_email_verification = false

### ISSUE THREAD

**Issue by centauromax** [userfrosting/UserFrosting#815]
Hi,
in the login form, the link "Resend verification email" is displayed even if site.registration.require_email_verification = false.

**Where is the problem**:

In the template userfrosting/app/sprinkles/account/templates/pages/sign-in.html.twig, the link is displayed without checking the value of the flag:

```
        <a href="{{site.uri.public}}/account/resend-verification">{{translate('ACCOUNT.VERIFICATION.RESEND')}}</a><br>
        {% if site.registration.enabled %}
            <a href="{{site.uri.public}}/account/register">{{translate('REGISTER')}}</a>
        {% endif %}
```

**Comment by alexweissman** [comment]
Actually, there is a use case where you'd want to still display this link, even if `site.registration.require_email_verification = false`.  This is the case where you turn off `require_email_verification` while there are still outstanding unverified users.

I guess the best thing to do, is check the value of `site.registration.require_email_verification` and ignore the user's verification status if it is set to `false`.

**Comment by alexweissman** [comment]
Ok, this is fixed in v4.1.14.
